jupyter_kernel_mgmt/subproc/manager.py

```diff
@@ -19,6 +19,13 @@
 
 from ..managerabc import KernelManagerABC
 
+# Abstract the decision on whether to use a sync popen so we
+# can easily enable it on non-windows systems.  This value should
+# remain True for win32 until asyncio.create_subprocess_exec works.
+if sys.platform == 'win32':
+    use_sync_subprocess = True
+else:
+    use_sync_subprocess = False
 
 class KernelManager(KernelManagerABC):
     """Manages a single kernel in a subprocess on this host.
```

Comment:
The docstring of `KernelManager` should be updated to clarify that it can have either a `Popen` or an asyncio `Process` object.